eng/sendToHelix.proj

```diff
@@ -34,8 +34,8 @@
   <ItemGroup Condition="'$(TestOS)' == 'Unix'">
     <!-- Xunit project to test -->
     <XUnitProject Include="..\src\System.Device.Gpio.Tests\System.Device.Gpio.Tests.csproj">
-      <TargetFramework>netcoreapp3.1</TargetFramework>
-      <RuntimeTargetFramework>netcoreapp2.0</RuntimeTargetFramework>
+      <TargetFramework>net6.0</TargetFramework>
+      <RuntimeTargetFramework>net6.0</RuntimeTargetFramework>
```

Comment:
```suggestion
      <RuntimeTargetFramework>netcoreapp2.0</RuntimeTargetFramework>
```

Reply:
Hey thanks, that appears to have worked (even though I'm confused as to why this requires such an old runtime).

Reply:
@pgrawehr if you check out the logs, you'll see it's running on runtime version 6.0.8:

```
xUnit.net Console Runner v2.4.2+f110e5bee5 (32-bit .NET 6.0.8)
  Discovering: System.Device.Gpio.Tests
```
...think of netcoreapp2.0 as a set of reference APIs ; newer runtimes support them in a roll-forward fashion.  Due to the challenges of naming things, this is a difficult subject to search.  In the case of this specific XUnit NuPkg library (you can run tests directly from "dotnet" now, alternatively), the only versions it's compiled for for .NET Core are netcoreapp1.0 and 2.0; any runtime which supports these API sets can still use the package, but not new surface area added in later netcoreapp* versions.

Reply:
right, the variable name is probably what is causing the confusion here. The runner only cross-compiles for a few runtimes, and net6.0 is not one of them, so all you are doing here is selecting which of those cross-compiled assets you want to use, but the actual runtime is the one you select with the other value, which is correctly using 6.0.8.